src/firstboot/firstboot.c

```diff
@@ -210,7 +210,7 @@ static int prompt_loop(const char *text, char **l, unsigned percentage, bool (*i
                         return log_error_errno(r, "Failed to query user: %m");
 
                 if (isempty(p)) {
-                        log_warning("No data entered, skipping.");
+                        log_debug("No data entered, skipping.");
```

Comment:
what's the rationale for this change?

Reply:
@keszybz mentioned this in #31154. The specific words used where 

> Stop saying "No data entered, skipping." at warning level. In particular in interactive mode this is not useful, the user knows that they pressed Enter.

Which I do kind of agree with. The only place I was somewhat unsure of is for the password.

Reply:
I think log_info() would still be good, to explain what goes on. Debug means it's hidden by default